src/stdio-bridge/stdio-bridge.c

```diff
@@ -82,11 +87,30 @@ static int parse_argv(int argc, char *argv[]) {
                 case '?':
                         return -EINVAL;
 
-                case 'p':
-                        arg_bus_path = optarg;
+                case 'p': {
+                        char *e = NULL;
+
+                        e = strdup(optarg);
```

Comment:
Then, now it is not necessary to duplicate `optarg`. And `arg_bus_str` can be `static const char *`.